uploader.json

```diff
@@ -2073,6 +2073,11 @@
             "pattern": [],
             "symbolic_name": "nfnetlink_queue"
         },
+        {
```

Comment:
@vishwanathjadhav this needs to be moved to the command section of the json instead of the files section.